x-pack/plugins/spaces/public/space_selector/space_selector.tsx

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import './_index.scss';
```

Comment:
Does this index file still need to be imported in this sass file anymore?

https://github.com/elastic/kibana/blob/501bec81b3583d648fcea81fe5a210ae05989fb7/x-pack/plugins/spaces/public/index.scss#L15